### PR TITLE
set max_stream_window_size=100M

### PR DESCRIPTION
**Pull request by freakinyy**
Optimize performance for high latency network environment. (#1)
According to
```
Bandwidth=((1000/t) * (MaxStreamWindowSize/2) * 8)/(1024*1024) Mbps
```
When latency t is 400ms and MaxStreamWindowSize is 100M,  the bandwidth is 1000Mbps in theory.

### CODE REVIEW

**Comment by zonyitoo** [comment]
Did you format the code with `rustfmt`?

**Comment by freakinyy** [comment]
No, just figured out what it is.
Can I format the code with ```vscode-rustfmt```?

**Comment by zonyitoo** [comment]
> 

If you are using vscode, the Rust language plugin should already builtin with `rustfmt`.

File: src/yamux-plugin-local.rs [hunk 1]
```diff
@@ -57,6 +57,11 @@ async fn get_yamux_stream(
     const YAMUX_CONNECT_RETRY_COUNT: usize = 3;
 
     let mut connect_tried_count = 0;
+
+    let custom_config = Config { 
+        max_stream_window_size: (100 * 1024 * 1024),
+         ..Config::default() };
```
Comment (zonyitoo) [inline src/yamux-plugin-local.rs]: Style format: alignment.

**Comment by freakinyy** [comment]
OK, I think code already formatted by nightly rustfmt.
Please check.

**Pull request merged**